fury/window.py

```diff
@@ -403,10 +406,13 @@ def initialize(self):
 
     def render(self):
         """Render only once."""
+        self.fps = 1.0 / (time.perf_counter() - self._last_render_time)
+        self._last_render_time = time.perf_counter()
         self.window.Render()
 
     def start(self):
         """Start interaction."""
+        # self._last_render_time = time.perf_counter()
```

Comment:
It seems this comment isn't required here.

Reply:
Yes, indeed. I was thinking about uncommenting it. But it won't do anything useful. So I'm going to remove it.